client/blocks/reader-related-card-v2/index.jsx

```diff
@@ -105,20 +94,42 @@ export function RelatedPostCard( { post, site, siteId, onPostClick = noop, onSit
 	const postClickTracker = partial( onPostClick, post );
 	const siteClickTracker = partial( onSiteClick, post );
 
+	const canonicalMedia = post.canonical_media;
+	let featuredAsset;
+	if ( ! canonicalMedia ) {
+		featuredAsset = null;
+	} else if ( canonicalMedia.mediaType === 'video' ) {
+		featuredAsset = <ReaderFeaturedVideo
```

Comment:
I should probably make a FeaturedAsset component that takes in a canonicalMedia and outputs the correct component